content/articles/how-to-secure-php-resource-with-http-authentication/index.md

```diff
@@ -0,0 +1,195 @@
+### How to secure PHP resource with HTTP authentication
+
+### Introduction
+
+HTTP authentication is the procedure of figuring out and verifying customers' credentials to ensure they are permitted to get the right of entry to an internet resource. They aim at preventing unauthorized entry to PHP web applications, by defending sensitive files or endpoints by the use of a username and a password or those containing Base64 encoded credentials. HTTP authentication helps significantly in increasing the speed of the process, security, revenue, and sensitive credentials of companies.
+
+### Outline
+
+1. [Editing the Apache configuration file](#editing-the-apache-configuration-file)
+
+2. [Enabling Apache `mod_rewrite` module](#enabling-apache-mod_rewrite-module)
+
+3. [Creating a password protected PHP file](#creating-a-password-protected-php-file)
+
+4. [Modifying the `.htaccess` file](#modifying-the-htaccess-file)
+
+5. [Testing HTTP Authentication](#testing-http-authentication)
+

```

Comment:
Have you noticed these spacings in-between your list? Kindly update this and any other applicable areas.

Reply:
@jumamiller i have fixed the errors sorry for the inconveniences caused.